File: Java/ReverseString_344.java
public class ReverseString_344 {
    public void reverseString(char[] s) {
        int len=s.length;
        int left=0,right=len-1;
        while(left<=right){
            char temp=s[left];
            s[left]=s[right];
            s[right]=temp;
            left++;
            right--;
        }
    }
    public static void main(String[] args) {
        ReverseString_344 obj=new ReverseString_344();
        char[] s={'h','e','l','l','o'};
        obj.reverseString(s);
        String reversed = new String(s);
        System.out.println("Reversed String: "+reversed);
    }
}

File: Python/ReverseString_344.py
from typing import List

class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        l=len(s)
        right=l-1
        left=0
        while left<=right:
            s[left],s[right]=s[right],s[left]
            left+=1
            right-=1
        return s
    
obj=Solution()
s=["h","e","l","l","o"]

print("Reversed string: ",obj.reverseString(s))